dev/com.ibm.ws.org.apache.cxf.rt.ws.security.3.4.1/src/org/apache/cxf/ws/security/wss4j/UsernameTokenInterceptor.java

```diff
@@ -0,0 +1,463 @@
+/**
```

Comment:
Some of these classes are copied to just get re-compiled , so that errors such as 
java.lang.NoSuchMethodError: org.apache.cxf.binding.soap.SoapMessage.put(Ljava/lang/String;Ljava/lang/Object;)Ljava/lang/Object;
        at org.apache.cxf.ws.security.wss4j.UsernameTokenInterceptor.storeResults(UsernameTokenInterceptor.java:190)
..

 (occurred due to our MessageImpl change) can be fixed. I updated the classes and included the comment.